crates/re_sdk_comms/src/server.rs

```diff
@@ -103,6 +103,12 @@ fn spawn_client(stream: TcpStream, tx: Sender<LogMsg>, options: ServerOptions) {
             re_log::info!("New SDK client connected: {addr_string}");
         }
         if let Err(err) = run_client(stream, &tx, options).await {
+            if let Some(err) = err.downcast_ref::<std::io::Error>() {
+                if err.kind() == ErrorKind::UnexpectedEof {
```

Comment:
should we also check `BrokenPipe` `ConnectionAborted` and `ConnectionReset`? I'm not 100% what is used under what circumstances

Reply:
Those wouldn't qualify as leaving _gracefully_, and thus aren't relevant for replacing `Goodbye`, but I agree we would still like to know about those in many cases.

It gets a bit more complicated due to the multiplexed nature of the smart channel, but I have a commit coming up which handles that among other things.